src/components/TodoInfo.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+type Props = {
+  status: boolean;
+  title: string;
+};
```

Comment:
https://www.typescriptlang.org/docs/handbook/utility-types.html
```suggestion
type Props = Pick<Todo, "title" | "status">
```